movies-app/src/components/MovieCard/MovieCard.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './MovieCard.scss';
+import CircleIcon from '../../core/constants/circle-icon.svg';
+import { MovieCardOptions } from '../MovieCardOptions';
+
+export const MovieCard = ({ movie }) => {
+  const { image, film, year, category } = movie;
+  const [isOptionsShown, setIsOptionsShown] = useState(false);
+  return (
+    <div className="movie-card">
+      <div className="movie-card-container">
+        <img
+          className="movie-card-container__image"
+          src={`core/constants/movieImages/${image}.png`}
```

Comment:
Consider extracting such logic into variables/functions out of JSX into component's body.